src/squid.md

```diff
@@ -0,0 +1,280 @@
+---
+title: squid
+section: 8
+---
+# NAME
+
+**squid** - HTTP web proxy caching server
+
+# SYNOPSIS
+
+**squid** [\-dhisrvzCFNRSVYX] [\--foreground] [\-l facility] [\-f config\-file] [\-[au] port] [\-k signal ] [\-n service\-name] [\-O command\-line]
+
+# DESCRIPTION
+
+**squid** is a high-performance proxy caching server for web
+clients, supporting FTP, ICAP, ICP, HTCP and HTTP data objects.
+Unlike traditional caching software, Squid handles all requests
+in a single, non-blocking process.
+
+Squid keeps meta data and especially hot objects cached in RAM,
+caches DNS lookups, supports non-blocking DNS lookups, and
+implements negative caching of failed requests.
+
+Squid supports TLS, extensive access controls, and full request
+logging.  By using the lightweight Internet Cache Protocols ICP,
+HTCP or CARP, Squid caches can be arranged in a hierarchy or
+mesh for additional bandwidth savings.
+
+Squid consists of a main server program **squid**, some optional
+programs for custom processing and authentication, and some
+management and client tools.  When squid starts up, it spawns a
+configurable number of helper processes, each of which can
+perform parallel lookups.  This reduces the amount of time the
+cache waits for results.
+
+Squid is derived from the ARPA funded Harvest Project.
+
+This manual page only lists the command line arguments. For
+details on how to configure Squid see the file
+**@SYSCONFDIR@/squid.conf.documented**, the Squid
+[wiki FAQ](https://wiki.squid-cache.org/) and examples, or the
+[configuration manual](http://www.squid-cache.org/Doc/config/).
+
+# OPTIONS
+
+-a port
+
+> Specify HTTP port number where Squid should listen for
+  requests, in addition to any
+  [http_port](http://www.squid-cache.org/Doc/config/html_port/)
+  in **@SYSCONFDIR@/squid.conf**
+
+-C
+
+> Do not catch fatal signals.
+
+-d level
+
+> Write debugging to stderr also.
+
+-f file
+
+> Use the given file instead of @SYSCONFDIR@/squid.conf .
+  If the file name starts with a **!** or **|** then it is
+  assumed to be an external command or command line. Can for
+  example be used to pre\-process the configuration before it is
+  being read by Squid. To facilitate this Squid also understands
+  the common #line notion to indicate the real source file.
+
+-F
+
+> Don't serve any requests until store is rebuilt.
+
+-h
+
+> Print help message.
+
+-i
+
+> Install as a Windows Service (see `-n` option).
+
+-k [ reconfigure | rotate | shutdown | interrupt | kill | debug | check | parse ]
+
+> Parse configuration file, then send signal to running copy
+  (except `-k parse`) and exit.
+
+-l facility
+
+> Use specified syslog facility. Implies `-s`
+
+-n name
+
+> Specify Windows Service name to use for service operations.
+  The default is **"Squid"**
+
+-N
+
+> No daemon mode.
+
+-\-foreground
```

Comment:
Just for my own edification, why do we need to escape the second dash here?

Reply:
IIRC the character `-` (minus/dash) is a special escaping character for the `man(N)` interpreter.

Without the slash the interpreter would convert the double-dash into a single hyphen-Long character. Which is both visually false (looks like a single dash), and cannot be cut-n-pasted by admin who do that.